### ISSUE TITLE
model.predict is very fast than model(inputs=tensor)

### ISSUE THREAD

**Issue by leekun** [keras-team/keras#10331]
I find "model.predict" is very fast than "model(inputs=tensor)". why? 
code : thank you thank you 

from keras.applications.mobilenet import MobileNet
from keras.applications.vgg19 import preprocess_input
from keras.preprocessing import image as kimage
import numpy as np
from keras import backend as K

mobile_net = MobileNet(weights='imagenet') # load model

dog_img = 'demo_pic/dog2.jpg' # load img
img = kimage.load_img(dog_img, target_size=(224, 224))
x = kimage.img_to_array(img)
x = np.expand_dims(x, axis=0)
x = preprocess_input(x) 

one_ret = mobile_net.predict(x)  # is fast

two_ret = mobile_net(inputs=K.variable(x))  # is very slow

**Comment by leekun** [comment]
That is a little weird... Thank you @Dref360 .
